db/books.cds

```diff
@@ -37,17 +36,21 @@ entity Authors : cuid, managed {
 
 // annotations for Data Privacy
 annotate Authors with
-@PersonalData : { DataSubjectRole : 'Author', EntitySemantics : 'DataSubject' }
-{
-  ID    @PersonalData.FieldSemantics : 'DataSubjectID';
-  name  @PersonalData.IsPotentiallySensitive;
+@PersonalData: {
+    DataSubjectRole: 'Author',
+    EntitySemantics: 'DataSubject'
+} {
+    ID @PersonalData.FieldSemantics: 'DataSubjectID';
+    name @PersonalData.IsPotentiallySensitive;
 }
 
 /**
  * Hierarchically organized Code List for Genres
  */
-entity Genres : sap.common.CodeList {
```

Comment:
Why can't you use CodeList from sap.common?

Reply:
@OlenaTi - do you recall why we changed this?

Reply:
-> https://github.com/SAP-samples/cloud-cap-samples-java/pull/419/files#diff-d92b8eb29957a03ad9d23b01feff79da293e30183d0b8eac5d893d95632e30dfR51

Reply:
https://github.com/SAP-samples/cloud-cap-samples-java/pull/388#issue-2624645140